test/38.database.hooks.js

```diff
@@ -0,0 +1,205 @@
+'use strict';
+const Database = require('../.');
+
+describe('Database hooks', function () {
```

Comment:
I'd like to see an additional test: attempting to use the database within a hook should fail, since the db is busy